x-pack/plugins/alerting_builtins/server/alert_types/geo_threshold/es_query_builder.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ILegacyScopedClusterClient } from 'kibana/server';
+
+export const OTHER_CATEGORY = 'other';
+
+async function getShapesFilters(
+  boundaryIndexTitle: string,
+  boundaryType: string,
+  boundaryGeoField: string,
+  geoField: string,
+  callCluster: ILegacyScopedClusterClient['callAsCurrentUser'],
+  log: Logger
+) {
+  const filters = {};
+  switch (boundaryType) {
+    case 'entireIndex':
+      // Get all shapes in index
+      const boundaryData = await callCluster('search', {
+        index: boundaryIndexTitle,
```

Comment:
I was curious what `indexId` and `indexTitle` were - I was thinking `indexId` might be the index pattern id, and `indexTitle` might be a name associated with that index pattern.  Although thinking about it, I think index pattern id's actually are the actual index pattern, and we provide the capability for customers to customize the id otherwise we use a uuid, for the actual SO id.

So, it was already kinda confusing, seems like this makes it a little more confusing, having it named this way.  I'm not familiar with how index pattern "variables" are typically named, perhaps this is kind of the standard anyway, and it's just me that would be confused :-)

Reply:
Agreed this is confusing. I've changed `indexTitle` to just be `index` to be consistent with code elsewhere in kibana. I've left `indexId` as is as I believe this variable name makes sense here